src/views/communities/components/AddCommunityDialog.js

```diff
@@ -0,0 +1,343 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
This is a low-priority task, but you can review how forms are implemented on the Registration form using Formlink. This will reduce the amount of code as validations could be written in a single line.https://github.com/salman299/react-communi-verse/blob/develop/src/views/pages/authentication/auth-forms/AuthRegister.js#L108

Reply:
Good Work!